sdk/cognitiveservices/Vision.Face.Preview/README.md

```diff
@@ -0,0 +1,55 @@
+# Azure Cognitive Services Face preview client library for .NET
+
+## Getting started
+
+### Install the package
+
+Install the Face client library for .NET with [NuGet][nuget]:
+
+```PowerShell
+dotnet add package Microsoft.Azure.CognitiveServices.Vision.Face.Preview --version  2.7.0-preview.2
+```
+
+### Prerequisites
+
+* An [Azure subscription][azure_sub].
+* An existing Cognitive Services or Face resource.
+
+### Authenticate the client
+
+To authenticate the FaceClient, you will need to provide the endpoint and API key for your Face resource.
+You can obtain the endpoint and API key from the resource information in the [Azure Portal][azure_portal].
+Instantiate the FaceClient with ApiKeyServiceClientCredentials as follows:
+
+```
+IFaceClient client = new FaceClient(new ApiKeyServiceClientCredentials({YourAPIKey}), handlers: handler)
+{
+    Endpoint = {YourEndpoint}
+};
+```
+
+## Key concepts
+
+### Thread safety
+We guarantee that all client instance methods are thread-safe and independent of each other ([guideline](https://azure.github.io/azure-sdk/dotnet_introduction.html#dotnet-service-methods-thread-safety)). This ensures that the recommendation of reusing client instances is always safe, even across threads.
+
+### Additional concepts
+
+## Examples
```

Comment:
We should consider filling these out, or ensuring that we have an issue open to do so in the near future.   The [Form Recognizer README](https://github.com/Azure/azure-sdk-for-net/tree/master/sdk/formrecognizer/Azure.AI.FormRecognizer) is a good example of content.